### PR TITLE
Velychko/feature/oos 76 should not be able go back past

### PR DESCRIPTION
**Pull request by maryna-velychko**

### CODE REVIEW

File: src/app/booking/OOSPlannerSection.js [hunk 1]
```diff
@@ -488,10 +487,8 @@ function BookingsTable() {
                 <Box sx={{ display: "flex", justifyContent: "space-between", alignItems: "center" }}>
                   <Typography variant="h6">{selectedDate.format("MMMM YYYY")}</Typography>
                   <Box sx={{ display: "flex", alignItems: "center", gap: 1 }}>
-                    <IconButton onClick={handlePreviosWeek}>
-                      <ArrowBackIosNewIcon />
-                    </IconButton>
-                    <Button color="grey" sx={{ py: 0.2 }} onClick={() => setSelectedDate(dayjs())}>
+                    <MoveNextWeekButton selectedDate={selectedDate} setSelectedDate={setSelectedDate}/>
+                    <Button color="grey" sx={{ py: 0.2 }} onClick={() => setSelectedDate(dayjs.utc())}>
```
Comment (shuryhin-oleksandr) [inline src/app/booking/OOSPlannerSection.js:491]: Extract to a separate PR

File: src/app/booking/MoveNextWeekButton.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+"use-client";
+
+import ArrowBackIosNewIcon from "@mui/icons-material/ArrowBackIosNew";
+import { IconButton, Box } from "@mui/material";
+import { getDisplayedWeekModayDate } from "./ActivitiesCalendar";
+import dayjs from "dayjs";
+
+export default function MoveNextWeekButton({ selectedDate, setSelectedDate }) {
+  const currentMonday = getDisplayedWeekModayDate(dayjs.utc())
+  const isLastWeek = getDisplayedWeekModayDate(selectedDate).isSame(currentMonday)
```
Comment (shuryhin-oleksandr) [inline src/app/booking/MoveNextWeekButton.js:10]: previousWeekButtonEnabled

File: src/app/booking/MoveNextWeekButton.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  const handlePreviosWeek = () => setSelectedDate(selectedDate.subtract(7, "day"));
+
+  return (
+    <Box>
+      {!isLastWeek &&
```
[the remark this answers is not in the document]
Comment (maryna-velychko) [reply src/app/booking/MoveNextWeekButton.js]: it is used by default
